src/main/kubernetes/artemis/pvc/artemis-mysql.yml

```diff
@@ -9,4 +9,4 @@ spec:
     requests:
       storage: 3Gi
   storageClassName: local-path
-  volumeMode: Filesystem
\ No newline at end of file
+  volumeMode: Filesystem
```

Comment:
Target specific. Move to kustomize file

Reply:
How is that target specific? According to the K8s [documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#volume-mode), it is supported by Kubernetes itself. Maybe I misunderstood you?

Reply:
You're absolutely right! I misunderstood the option - Sorry! My bad